### DISCUSSION TITLE
Azure / Docker / Media conversions Could not load image at path

### DISCUSSION THREAD

**Discussion by danharper83** [spatie/laravel-medialibrary#3611] [Q&A]
I have deployed an application to Azure using azure pipelines and docker but I'm getting an error when Spatie tries to create the conversions.

Could not load image at path `/var/www/html/storage/media-library/temp/DlzYfIFNSki8m9f3rA5KorkYF8vKLBv1/ILf5uwHw9rMKYPO9NL0x6qNmlg5Gvr9bpreview.jpg` at /var/www/html/vendor/spatie/image/src/Exceptions/CouldNotLoadImage.php:11

If I ssh into the container I can see that the image does exist in this path so I guess for some reason it can't read / see the iamge in that directory. The main image does actually get created and I can see it in my Laravel application.

I'm using GD Image and it is enabled in PHP.

Any help troubleshooting would be appreciated.

Cheers Dan

**Comment by bhaidar** [comment]
@danharper83 I am facing something similar on Digital Ocean spaces. I can open the image from the DO Spaces but when I try to download it it says doesn't exist. If I stream it, I get damaged image.

https://github.com/spatie/laravel-medialibrary/discussions/3610

**Comment by paaneko** [comment]
Same issue, when I am trying to run migrations in docker

**Comment by ahmedshahd** [comment]
same issue here with aws ec2 and docker

**Comment by abezruchenkov** [comment]
I had the same issue and it appeared that my php didn't support webp format. After enabling webp in php gd this problem disappeared.

Try to debug your path like this:

$path = 'your_path';
$handle = fopen($path, 'r');
$contents = fread($handle, filesize($path));
fclose($handle);

// in most cases exception will be trown here 
// but in original code this line is suppressed with @ 
$image = imagecreatefromstring($contents);
